hal_st/stm32fxxx/CMakeLists.txt

```diff
@@ -75,6 +77,8 @@ target_sources(hal_st.stm32fxxx PRIVATE
     SystemTickStm.hpp
     SystemTickTimerService.cpp
     SystemTickTimerService.hpp
+    $<$<STREQUAL:${TARGET_MCU},stm32wb55>:TimerStm.cpp>
+    $<$<STREQUAL:${TARGET_MCU},stm32wb55>:TimerStm.hpp>
```

Comment:
If it compiles for every target, and if it runs on at least one target, we assume it's great for every target

```suggestion
    TimerStm.cpp
    TimerStm.hpp
```